nvflare/edge/widgets/evaluator.py

```diff
@@ -65,9 +69,13 @@ def __init__(
         self.model = None
         self.data_loader = None
         self.tb_writer = None
+        self.max_workers = max_workers
+
+        self._thread_pool = None
```

Comment:
You can create the thread_pool here - no need to do it later since it doesn't depend on other things.

Reply:
moved here